doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='interacting-with-processes-process-instances-default-filters-proc']
+= Filtering process instances using default filters
```

Comment:
Trying to think a better way to describe the actual feature, maybe something like: Select a default filter for Manage Process Instances, wdyt?

Reply:
Ack